packages/shared-ux/page/solution_nav/src/solution_nav.tsx

```diff
@@ -110,8 +105,8 @@ export const KibanaPageTemplateSolutionNav: FunctionComponent<
   const isHidden = isLargerBreakpoint && !isOpenOnDesktop && canBeCollapsed;
   const isCustomSideNav = !!children;
 
-  const sideNavClasses = classNames('kbnPageTemplateSolutionNav', {
-    'kbnPageTemplateSolutionNav--hidden': isHidden,
+  const sideNavClasses = classNames('kbnSolutionNav', {
```

Comment:
Do we need this `kbn` prefix here? I believe `kbn` came from `kbnPageTemplate`; since we're getting rid of that, we can remove the `kbn` part as well?

Reply:
We need a three-letter prefix to ensure we scope styles properly. I'm up for discussing if we need a shared-ux specific prefix, but `kbn` also feels appropriate as it's generally a Kibana-wide component.

Reply:
Absolutely.  Will refactor.

Reply:
Good point, @cchaos... I'll keep the `kbn` prefix.